### ISSUE TITLE
Different position in database and in traccar-web

### ISSUE THREAD

**Issue by gsiotas** [vitalidze/traccar-web#1171]
![screen shot 2017-09-15 at 14 00 54](https://user-images.githubusercontent.com/10040858/30479733-3fe532b0-9a1f-11e7-8952-a57605abc363.png)

I have a strange problem. In the database (and in the log file) I can see that the device latest send position is "2017-09-15 13:56:17" (green arrow) but at the pop-up message the latest position of the device is 6 days ago (red arrow). Also in the archive panel I have the whole history of the device for today. For security reason I have blurred IP and GPS coordinates.

Any ideas that may cause this bug? Also I have checked the database positions table for invalid positions in the future that may cause traccar-web to not correctly display the position, but there aren't such positions.

Thank you!

**Comment by gsiotas** [comment]
Also I would like to mention that the latest position since the device seems idle is (2016-11-01 18:03:49) as you can see from the popup message in the screenshot. However this position can't be found in the positions table in the database. Is there somewhere else that the positions are stored (either database or file)?

**Comment by vitalidze** [comment]
I have no idea why it worked this way. Have you tried restarting the service?

**Comment by gsiotas** [comment]
Yes I have tried restarting the service with no success. Fortunately I have found somewhat of a solution. I duplicated the device with all the points, deleting points from time to time. Somewhere along the way it worked fine. After further investigation I found that responsible for that peculiar bug was a particular point. I deleted it and everything worked flawless. I don't know why this particular point was causing an error. Anyway I'm closing this and if this error appears again I'll  notify again.

Thank you
